sdcm/db_stats.py

```diff
@@ -453,6 +453,10 @@ def get_scylla_versions(self):
         versions = {}
         try:
             node = self.db_cluster.nodes[0]
+            if node.scylla_version_detailed and 'build-id' in node.scylla_version_detailed:
+                build_id = node.scylla_version_detailed.split('with build-id ')[-1]
```

Comment:
better to strip() too to eliminate possoble new lines and other whitespaces

Reply:
added